site/content/api/Event-API.md

```diff
@@ -18,7 +18,7 @@ The following operations can be used by *Devices* and/or *Protocol Adapters* (to
 
 Both *Devices* as well as *Protocol Adapters* will be referred to as *clients* in the remainder of this section.
 
-## Send Event
+## Upload Event Data
```

Comment:
Why changing from send to upload ?

Reply:
I just used the same phrase as already used for the telemetry page. Do you think "sending events" and "uploading telemetry data" would be better (as it was before)? I was not quite sure myself and then decided for that... WDYT?